chef/cookbooks/postgresql/templates/default/pgsql.sysconfig.erb

```diff
@@ -1,4 +1,40 @@
-PGDATA=<%= node['postgresql']['dir'] %>
-<% if node['postgresql']['config'].attribute?("port") -%>
-PGPORT=<%= node['postgresql']['config']['port'] %>
```

Comment:
PGPORT is not needed any more. Attribute `node['postgresql']['config']['port']` is used for postgres.conf file